Update robot-food-time.md

Vairus opened the pull request:
Changed make_a_grill_cheese_sandwich function.
Changed recipe.

enki-hq commented:
### Insight previews:
- [ ] [enkidevs/curriculum/coding-intro/core/functions/robot-food-time.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/coding-intro/core/functions/robot-food-time.md?ref=dcfdbb3653020e306a8ced464e730eab0c3761db)

Vairus commented:
Changed make_a_grill_cheese_sandwich function.
Changed recipe.
If approved, I will change the recipe in the rest of the pages too.
Someone has mentioned in the comments that knife should be added to the utensils.
I think it's correct, since grater is mentioned as well.
If approved, then the prepare_ingredients function should be changed as well.

kapnobatai137 commented:
Hey @Vairus, great suggestion :muscle: 

As you have mentioned, we need to make a couple more changes to include the `knife` in the code.

For starters, in the `Robot, I'm Hungry!` insight ([here](https://github.com/enkidevs/curriculum/blob/master/coding-intro/core/functions/robot-im-hungry.md)) we should update the recipe footnote (the change should be the same as the one you already made) and update the code block to include the knife:

```plain-text
FUNCTION prepare_ingredients
  INPUT cheese, grater, bowl, bread, knife
  INSTRUCT cheese_bowl = grate_cheese_in_the_bowl(grater, cheese, bowl)
  INSTRUCT sliced_bread = slice_the_bread(bread, knife)
  OUTPUT cheese_bowl, sliced_bread
ENDFUNCTION
```

In the next insight (`Robot, Assemble the Sandwich`, [here](https://github.com/enkidevs/curriculum/blob/master/coding-intro/core/functions/robot-assemble-the-sandwich.md)), we should also update the recipe footnote, and update the code block to use the knife as well:

```plain-text
FUNCTION build_sandwich
  INPUT cheese_bowl, sliced_bread, butter, knife
  INSTRUCT buttered_slices = butter_the_bread(sliced_bread, butter, knife)
  INSTRUCT buttered_slices_with_cheese = add_the_cheese_to_each_slice(cheese_bowl, buttered_slices)
  INSTRUCT sandwich = combine_slices(buttered_slices_with_cheese)
  OUTPUT sandwich
ENDFUNCTION
```

In the following insight (`Robot, Grill Me A Sandwich`, [here](https://github.com/enkidevs/curriculum/blob/master/coding-intro/core/functions/robot-grill-me-a-sandwich.md)), we should update the recipe footnote.

Lastly, in the insight that you have edited (`Robot, Food Time`), let's make the code block like this:

```plain-text
FUNCTION make_a_grill_cheese_sandwich
  INPUT cheese, grater, bowl, bread, butter, pan, plate, knife
  INSTRUCT cheese_bowl, sliced_bread = prepare_ingredients(cheese, grater, bowl, bread, knife)
  INSTRUCT sandwich = build_sandwich(cheese_bowl, sliced_bread, butter)
  INSTRUCT sandwich_on_plate = grill_sandwich(sandwich, pan, plate)
  OUTPUT sandwich_on_plate
ENDFUNCTION
```

We need to input the `knife` in the `prepare_ingredients` function because it now requires that parameter (after we edited the function). Also, we want to save the output of the function in variables (e.g. `cheese_bowl`, `sliced_bread`, `sandwich`, and `sandwich_on_plate`) so that we can use these variables as inputs for the other functions.

Let me know if you want to handle these yourself, or if you want me to take over.

Once again, great contribution. Thank you :fist_raised:
